torch/csrc/jit/python/python_tree_views.cpp

```diff
@@ -104,7 +104,9 @@ void initTreeViewBindings(PyObject* module) {
             return SourceRange(self.source_, start, end);
           })
       .def_property_readonly("source", [](const SourceRangeFactory& self) {
-        return self.source_->text();
+        auto text_view = self.source_->text();
+        std::string text = std::string(text_view.begin(), text_view.end());
```

Comment:
nit: this can be more succinctly written as `std::string text(text_view.begin(), text_view.end());`